project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.17
+sbt.version=0.13.18
```

Comment:
can this be bumped to something like `sbt.version=1.3.10`

Reply:
Good call. Implemented